packages/web-components/fast-foundation/src/text-field/text-field.template.ts

```diff
@@ -2,6 +2,17 @@ import { html, ref, slotted } from "@microsoft/fast-element";
 import { endTemplate, startTemplate } from "../patterns";
 import { TextField } from "./text-field";
 
+/***
+ * A simple whitespace filter for the template
+ * @public
+ */
```

Comment:
Can you add types to the function arguments?

Reply:
Updated